### PR TITLE
MGDAPI-3019 Update Readme with rhoam uninstall documentation

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**
# Issue link
https://issues.redhat.com/browse/MGDAPI-3019

# What
Addition a new piece of documentation explaining different ways of uninstalling RHOAM.

# Verification steps
- Make sure it follows the same convention as the rest of the documentation and makes sense when reading.
- Optional can verify that all the different uninstallation methods work.

### CODE REVIEW

**Comment by trepel** [comment]
I have read through it, looks good. I did not actually try the commands but I compared them with my personal notes and with what's in the pipeline code and everything matches so it must be ok. Gmail spell checker did not discover any typo either.

/lgtm

File: README.md [hunk 1]
```diff
@@ -223,6 +223,48 @@ To run products tests against an existing RHMI cluster:
...
+```sh 
+oc delete rhmi rhoam -n redhat-rhoam-operator
+```
+
+In both scenarios wait until the RHOAM cr is removed and then run the following command to delete the namespace.
+```sh 
+oc delete namespace redhat-rhoam-operator
+```
+
+#### Note : After uninstalling RHOAM you should clean up the cluster by running the following command.
```
Comment (MStokluska) [inline README.md:247]:
```suggestion
#### Note: After uninstalling RHOAM you should clean up the cluster by running the following command.
```

File: README.md [hunk 2]
```diff
@@ -223,6 +223,48 @@ To run products tests against an existing RHMI cluster:
...
+```sh
+export INSTALLATION_TYPE=managed-api
+make cluster/cleanup && make cluster/cleanup/crds
+```
+
+### Addon
```
Comment (MStokluska) [inline README.md:253]: Do we need to include addon installation here? If we are not mentioning it in `installation` section, I would not mention it in removal

File: README.md [hunk 3]
```diff
@@ -223,6 +223,48 @@ To run products tests against an existing RHMI cluster:
 make test/products/local
 ```
 
+## Uninstalling RHOAM
+
```
Comment (MStokluska) [inline README.md:227]: This section covers 
```suggestion
This section covers uninstallation of RHOAM if it was installed via locally, OLM or on ROSA
```

File: README.md [hunk 4]
```diff
@@ -223,6 +223,48 @@ To run products tests against an existing RHMI cluster:
...
+### Local and Catalog Source
```
Comment (MStokluska) [inline README.md:228]:
```suggestion
### Local and OLM installation type
```

File: README.md (hunk 2 above)
Comment (Patryk-Stefanski) [reply README.md:253]: Since that section is relatively small I think it's no harm to leave it in.

**Pull request merged**